common/utils.py

```diff
@@ -67,8 +67,39 @@ def make_response(status_code, body, header=None):
     }
 
 
-def validate_request(required_keys, request_body):
+def validate_dict(item, required_keys):
```

Comment:
@prashantramangupta @pratik-vii   this function is removed are we sure we are not using it anyehre else

Reply:
we are using method validate_dict(data_dict, required_keys)